playbooks/templates/.env.j2

```diff
@@ -95,41 +118,59 @@ DISCORD_MENTION_USER_ID={{ discord_mention_user_id | default('') }}
 # S3 backup path.
 # For prod servers: {{ webportal_common_config.s3_backup_path }}
 # For dev servers: {{ webportal_common_config.s3_backup_path }}_<server hostname>
+#
+# OPTIONAL: manually defined by user, can be blank
 S3_BACKUP_PATH={{ webportal_common_config.s3_backup_path }}{{ '_' + inventory_hostname if 'webportals_prod' not in group_names else '' }}
 
 # Used to load skylinks blocklist data from Airtable
-AIRTABLE_API_KEY={{ webportal_common_config.airtable_api_key }}
-AIRTABLE_BASE={{ webportal_common_config.airtable_base }}
-AIRTABLE_TABLE={{ webportal_common_config.airtable_table }}
-AIRTABLE_FIELD={{ webportal_common_config.airtable_field }}
+#
+# OPTIONAL: manually defined by user, can be blank
+AIRTABLE_API_KEY={{ webportal_common_config.airtable_api_key | default('') }}
+AIRTABLE_BASE={{ webportal_common_config.airtable_base | default('') }}
+AIRTABLE_TABLE={{ webportal_common_config.airtable_table | default('') }}
+AIRTABLE_FIELD={{ webportal_common_config.airtable_field | default('') }}
 
 ###############################################################################
 # MongoDB
 
 # MongoDB username
+#
+# REQUIRED: manually defined by user, or default will be used
 SKYNET_DB_USER={{ mongodb_config.skynet_db_user }}
 
 # MongoDB password
+#
+# REQUIRED: manually defined by user, or Ansible automatically generated 
 SKYNET_DB_PASS={{ mongodb_config.skynet_db_pass }}
 
 # MongoDB address or container name
-SKYNET_DB_HOST=mongo
+#
+# REQUIRED: manually defined by user, or default will be used
+SKYNET_DB_HOST={{ mongodb_config.skynet_db_host }}
```

Comment:
Is this configurable for all services using mongo?
Isn't `mongo` container service/host name hardcoded in some docker services using mongo?

Reply:
There are two hardcoded references 

https://github.com/SkynetLabs/skynet-webportal/blob/master/docker/malware-scanner/Dockerfile#L16
https://github.com/SkynetLabs/skynet-webportal/blob/master/docker/accounts/Dockerfile#L16

But then they are overwritten in the docker-compose files

https://github.com/SkynetLabs/skynet-webportal/blob/master/docker-compose.accounts.yml#L40

So since nothing is hardcoded in the docker compose files, this means that they are configurable right?